data/org.freedesktop.portal.ScreenCast.xml

```diff
@@ -119,6 +119,37 @@
               This option was added in version 2 of this interface.
             </para></listitem>
           </varlistentry>
+          <varlistentry>
+            <term>restore_token s</term>
+            <listitem><para>
+              The token to restore a previous session.
+
+              If the stored session cannot be restored, this value is ignored
+              and the user will be prompted normally. This may happen when, for
+              example, the session contains a monitor or a window that is not
+              available anymore, or when the stored permissions are withdrawn.
+
```

Comment:
As with the documentation for the token itself, maybe add something about the inability to reuse the same token again after restoring once. E.g.

```
After having used the restore token once, it will no longer be valid.
To restore the same session again, use the new restore token sent
in response to starting this session.
```

or something.

Reply:
Added